website/docs/releases/2025/v2025.2.md

```diff
@@ -13,6 +13,12 @@ slug: "/releases/2025.2"
 
 ## Breaking changes
 
+- **Fixed behaviour in Source stage <span class="badge badge--primary">Enterprise</span>**
+
+    In previous versions, the Source stage would incorrectly continue with the initial flow after returning from the source, which didn't match the documented behaviour.
```

Comment:
What exactly does "returning form the source" mean"? Does this mean after retrieving credentials for authnz from the source, or...?

Reply:
after the source redirects back to authentik

Reply:
OK, LGTM, I think most will understand it but I need to learn more.